andela/ah-sealteam-frontend
# Feature/users should be able to read an article 162694516

#### What does this PR do?
```Creates functionality for users to view specific articles ```
#### Description of Task to be completed?
```A user should be able to view an article```
#### How should this be manually tested?
```
$ git fetch origin pull/19/head:feature/Users-should-be-able-to-read-an-article-162694516
$ git checkout feature/Users-should-be-able-to-read-an-article-162694516
$ source .env
$ npm install
$ npm start
```
#### What are the relevant pivotal tracker stories?

[#162694516](https://www.pivotaltracker.com/story/show/162694516)
 
#### Checklist:

- [ ] At least 4 people have reviewed my PR

- By @botnetdobbs and @gtsofa
